packages/cozy-stack-client/src/DocumentCollection.js

```diff
@@ -354,6 +354,12 @@ class DocumentCollection {
       bookmark,
```

Comment:
```suggestion
      bookmark: options.bookmark || bookmark,
```

plutôt que la mutation conditionnelle plus bas